ansible/roles/ckan/templates/ckan.ini.j2

```diff
@@ -43,6 +43,7 @@ ckan.datastore.read_url = postgresql://{{ item.db_ckan_datastore_username }}:{{
 
 ckan.redis.url = redis://{{ ckan_redis_host }}:{{ ckan_redis_port }}/{{ item.redis_db }}
 
+ckan.emails.admin = {{ ckan_admin_email }}
```

Comment:
This should be named as extension like ckanext.organizationapproval.admin_email

Reply:
I was thinking that it might also be useful to use in other extensions for email sending, but I can change it to that if it's better.

Reply:
At least prefix it with ckanext instead of ckan since it is not a core configuration.

Reply:
Yeah. I moved it and decided to change it to `ckanext.organizationapproval.admin_email` now that it is a separate module and repository.